src/client/components/MessageForm/SendMessageForm.js

```diff
@@ -27,14 +27,19 @@ function SendMessageForm({ channelId, userId }) {
   return (
     <>
       <form onSubmit={handleSubmit} className="send-new-message-form">
-        <img
-          src="https://upload.wikimedia.org/wikipedia/commons/thumb/8/8d/List-Icon.svg/768px-List-Icon.svg.png"
-          alt=""
-        />
-        <input
-          value={input}
-          onChange={(event) => setInput(event.target.value)}
-        />
+        <div className="new-message-form-imag">
+          <img
+            src="https://upload.wikimedia.org/wikipedia/commons/thumb/8/8d/List-Icon.svg/768px-List-Icon.svg.png"
+            alt=""
```

Comment:
Always add a readable alt caption.

Reply:
I worked only on CSS part not javascript for example adding alt. It was there , like that, but i will fix that also.